src/functional/ffront/decorator.py

```diff
@@ -300,6 +317,10 @@ def _validate_args(self, *args, **kwargs) -> None:
             ) from err
 
     def _process_args(self, args: tuple, kwargs: dict) -> tuple[tuple, tuple, dict[str, Any]]:
+        # if parameter is in signature but not in args, move it from kwargs to args
+        if len(kwargs) > 0:
+            args, kwargs = _canonicalize_args(self.past_node.params, args, kwargs)
```

Comment:
```suggestion
        args, kwargs = _canonicalize_args(self.past_node.params, args, kwargs)
```
Either skip the check or move it to `_canonicalize_args`.